components/update_client/BUILD.gn

```diff
@@ -246,6 +246,7 @@ target(gtest_target_type, "cobalt_slot_management_test") {
     "//starboard/loader_app:installation_manager",
     "//testing/gmock",
     "//testing/gtest",
-    "//starboard/loader_app"
   ]
+
+  data_deps = ["//starboard/loader_app($starboard_toolchain)",]
```

Comment:
I get these errors 

+ gn gen /__w/cobalt/cobalt/out/evergreen-x64_devel '--args=target_platform="evergreen-x64"   target_cpu="x64" use_asan=false is_internal_build=false build_type="devel"'
Build Accelerator: sccache
Build Accelerator: sccache
Done. Made 740 targets from 282 files in 981ms
+ gn check /__w/cobalt/cobalt/out/evergreen-x64_devel
Build Accelerator: sccache
Build Accelerator: sccache
ERROR at //components/update_client/cobalt_slot_management_test.cc:26:11: Include not allowed.
#include "starboard/loader_app/system_get_extension_shim.h"
          ^-----------------------------------------------
It is not in any dependency of
  //components/update_client:cobalt_slot_management_test
The include file is in the target(s):
  //starboard/loader_app:loader_app
which should somehow be reachable.
___________________
ERROR at //starboard/loader_app/system_get_extension_shim.cc:15:11: Include not allowed.
#include "starboard/loader_app/system_get_extension_shim.h"
          ^-----------------------------------------------
It is not in any dependency of

when i make it a data_dep


should i keep it as a dep for now ? note that this blocks the 24lts cp's which is what im trying to solve

Reply:
This is an issue that should be fixed, but it can be done in a follow up